app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_posts_counter
+
+  private
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] I got the error message screenshot below as a result of the new implementation you did, I think this is because  you called a different method. Kindly implement it.


<img width="564" alt="Untitled" src="https://github.com/zhorabay/Blog/assets/103386183/f11cc0b8-88a3-4a2f-ac66-3d51eaa027f4">

```suggestion

  after_save :update_comments_counter

  private

  def update_comments_counter
    post.update(comments_counter: post.comments.count)
  end
```

**Note: Your app was running smoothly as of last review, the reason for this change is as a result of the code you newly added and resulted into and error **